lib/ui/locations/infoj.mjs

```diff
@@ -114,7 +114,7 @@ function entryDefault(entry) {
 
 function entryTitle(entry) {
 
-  if (entry.title) return;
+  if (!entry.title) return;
```

Comment:
Noticed an issue here where if the entry had a title it would return. 

Was just missing the operator.